package.json

```diff
@@ -99,13 +99,13 @@
     "zone.js": "0.15.1"
   },
   "devDependencies": {
-    "@angular-builders/custom-esbuild": "^19.1.0",
-    "@angular-devkit/build-angular": "19.0.7",
-    "@angular/cli": "19.0.6",
-    "@angular/compiler": "19.0.6",
-    "@angular/compiler-cli": "19.0.6",
-    "@angular/language-service": "19.0.6",
-    "@angular/localize": "19.0.6",
+    "@angular-builders/custom-esbuild": "20.0.0",
+    "@angular-devkit/build-angular": "20.1.1",
+    "@angular/cli": "20.1.1",
+    "@angular/compiler": "20.1.2",
```

Comment:
@spliffone Sorry I told you that we can go to the latest here but this is actually wrong. We tried to stay on the `.0` version to make sure we do not accidentally introduce new features that would not be backwards compatible. So please go back to `20.0`. Thank you 🙏